remapped floating islands

teoman002 opened the pull request.

bretonium commented:
could you please post a screenshot? I know that Kudonix has remade the map too, maybe he could comment on your version too

teoman002 commented:
![Floating_Island_v2_pic](https://user-images.githubusercontent.com/1367821/70869852-d8b36100-1f84-11ea-8858-4c6c3972c2a0.png)

teoman002 commented:
We already have decided that his upload has priority! Merging of both maps is impossible.

TheVayt commented:
Well I was going to negotiate with the map author (Libretee) first

Kaffeine commented:
(I hope you added proper background colors to the bonus zone(s))

TheVayt commented:
![screenshot_2019-12-16_18-35-40](https://user-images.githubusercontent.com/31164554/70921562-94cc6480-2035-11ea-9b13-e9039ba5a5c0.png)
![screenshot_2019-12-16_18-32-44](https://user-images.githubusercontent.com/31164554/70921571-985feb80-2035-11ea-85ff-a73622c39d8f.png)

teoman002 commented:
I updated my map. Please include it from yavl's master branch.
I also added .mapinfo files for both maps to adjust the play time.

![Screenshot from 2020-06-07 21-06-05](https://user-images.githubusercontent.com/1367821/83977695-07861080-a8f2-11ea-8c0d-a41208ceba70.png)
